test/arrays/vass/const-checking.aliases.good

```diff
@@ -1,9 +1,2 @@
-const-checking.aliases.chpl:8: In function 'main':
-const-checking.aliases.chpl:10: error: illegal lvalue in assignment
-const-checking.aliases.chpl:12: error: illegal lvalue in assignment
-const-checking.aliases.chpl:14: error: creating a non-const alias 'aa2' of a const array or domain
-const-checking.aliases.chpl:18: error: creating a non-const alias 'dd2' of a const array or domain
-const-checking.aliases.chpl:23: error: illegal lvalue in assignment
-const-checking.aliases.chpl:24: error: illegal lvalue in assignment
-const-checking.aliases.chpl:27: error: illegal lvalue in assignment
-const-checking.aliases.chpl:28: error: illegal lvalue in assignment
```

Comment:
I think we're saying similar things: that because the compiler exits prior to hitting the pass that prints the errors, we don't hit them anymore.  This is what I was suggesting in cloning the test with those lines commented out:  We get checks that both kinds of conditions generate errors, whereas by accepting this .good without the clone, we lose coverage of certain error cases that this test was locking in before (which may also be covered by other tests, but who knows...?).